app/controllers/api/ims_exports_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::ImsExportsController < ApplicationController
```

Comment:
Could you extend ApiApplicationController instead? That would let you not have to do the respond_to stuff in every method.

Reply:
No. I don't want the code in ApiController since we have to do a verification of a jwt from Canvas